app/view/set-password.ejs

```diff
@@ -0,0 +1,49 @@
+<% include partials/head.ejs %>
+<title>Set password | Drinkerz</title>
+</head>
+
+<body>
+  <main class="sign-up__container">
+    <section class="sign-up__section">
+      <figure class="sign-up__header">
+        <img src="/static/media/images/logo.png" alt="logo Drinkerz">
+        <h1 class="sign-up__title">Drinkerz</h1>
+      </figure>
+
+      <form class="sign-up__form" action="/set-password" method="POST">
+        <p>
+          <%- user.username %>
+        </p>
+
+        <label>Password</label>
+        <input type="password" name="password" placeholder="password">
+
+        <label>Gender</label>
+        <input type="radio" name="gender" value="male"> Male<br>
+        <input type="radio" name="gender" value="female"> Female<br>
+        <input type="radio" name="gender" value="other"> Other
+
+        <label>Age</label>
+        <input type="text" name="age" placeholder="age">
+
+        <p>preferred matches</p>
+
+        <fieldset class="preferred__age">
+          <label>Set age range</label>
+          <input type="text" name="age_min" placeholder="20">
+          <p>till</p>
+          <input type="text" name="age_max" placeholder="30">
+        </fieldset>
+
+        <label>Likes</label>
+        <input type="radio" name="preferred_gender" value="male"> Male<br>
+        <input type="radio" name="preferred_gender" value="female"> Female<br>
+        <input type="radio" name="preferred_gender" value="both"> both
+        <button class="button--fullwidth" type="submit">Submit change</button>
```

Comment:
Use __ instead of -- at class="button--fullwidth"

Reply:
@markschuttehva No need to if the `--fullwidth` is a modifier of the button element. See http://getbem.com/naming/.